graphx/src/main/scala/org/apache/spark/graphx/GraphOps.scala

```diff
@@ -411,8 +411,8 @@ class GraphOps[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED]) extends Seriali
    *
    * @see [[org.apache.spark.graphx.lib.ConnectedComponents$#run]]
    */
-  def connectedComponents(): Graph[VertexId, ED] = {
-    ConnectedComponents.run(graph)
+  def connectedComponents(numIter: Int = Int.MaxValue): Graph[VertexId, ED] = {
```

Comment:
you are right, I have fixed it. Now there is another ConnectedComponents API with numIters